data-es2016plus.js

```diff
@@ -3287,6 +3294,7 @@ exports.tests = [
         */},
         res : {
           closure20190215: true,
+          babel7corejs2: true,
```

Comment:
The JSON superset is supported by [@babel/proposal-json-strings](https://babeljs.io/docs/en/babel-plugin-proposal-json-strings) and included in `preset-env`, although by default the `eval`-ed strings will not be transpiled.

Note that we could transpile them with the help of https://github.com/babel/babel-test262-runner/blob/master/lib/run-tests/plugins/transform-eval.js but I feel like it's a bit overkill here especially the compiler test page is only one of reference.